My branch

shefali12-ab opened the pull request:
Added the required content in page opensource_programs.adoc
![gitbasic1](https://user-images.githubusercontent.com/56781823/135845925-bda75559-162b-4ac1-bb62-cb6f580798b8.png)
![gitbasic3](https://user-images.githubusercontent.com/56781823/135845931-f4b1c1dd-765f-4c00-bf62-cf5fba4986d8.png)
![gitbasic5](https://user-images.githubusercontent.com/56781823/135845934-0c6769ca-3868-492a-8129-e0bd4ff1b577.png)
![gitbsc2](https://user-images.githubusercontent.com/56781823/135845941-a6cc4ddf-0cb8-4d9c-8d31-d496f40f1d0f.png)
![gitbsc4](https://user-images.githubusercontent.com/56781823/135845943-83d8b799-a2cc-4862-b779-485aa1773563.png)
I have added these changes in new branch named my_branch. this much content i will change font style and format text afterwards.
please review it .

shefali12-ab commented:
@KartikSoneji and @HarshKapadia2 please have a look .

KartikSoneji commented:
This is a proper draft pull request, good job!
Please add `Closes #<issue number>` to the pull request description, that way the issue will automatically be closed when this PR is merged.
Also, it seems like the build step has some errors, note that you will need to fix them before the PR can be merged.

shefali12-ab commented:
okay @KartikSoneji where to see these errors. i am geeting mail regarding github page workflow failed. 
![eror](https://user-images.githubusercontent.com/56781823/135888733-8cd4a31f-8539-4695-80b1-e9b41543b2a7.png)

shefali12-ab commented:
@KartikSoneji is build setup error has been resolved ?? also is this much content is ok? some of the things I have written by my own so please check. also one ques as this repo is part of Hacktoberfest can this issue get a label of hacktoberfest?

HarshKapadia2 commented:
The workflow has been updated to handle PRs. [Please update your branch with the latest commits.](https://harshkapadia2.github.io/git_basics/#_sync_a_fork_with_its_original_repo)

Also, you have used syntax arbitrarily. For bullet points, please use `*` and `====` for headings. (The number of `=` can change.)
Please do not leave lines and whitespaces arbitrarily.
Please refer to other files in the project, the [CONTRIBUTING.md file](https://github.com/HarshKapadia2/git_basics/blob/main/CONTRIBUTING.md#language-code-and-syntax) and the [Asciidoctor syntax reference](https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference) for the code style and acceptable syntax.

shefali12-ab commented:
okay, sure I will do it.

shefali12-ab commented:
@KartikSoneji @HarshKapadia2 
I have added the required changes,
in place of unofficial sites I have added some blog posts.
also added logo images, Eligibility Section.
please check that list element section for combining two list elements I have just added + at the end of each para, is this correct? as I have read in the documentation that + is added in the new line, but that was not working here so, I  have added it at the end of para.

shefali12-ab commented:
@KartikSoneji @HarshKapadia2  Please Review it.

HarshKapadia2 commented:
Hey @shefali12-ab I would recommend that you try to build the web site. You have not linked the files properly, so it is not appearing in the final build. Please read the ['Parsing' section](https://github.com/HarshKapadia2/git_basics/blob/main/CONTRIBUTING.md#parsing) in the `CONTRIBUTING.md` file to understand how to link files.

Please test the final site before pushing.

You have a typo in the file name as well.

So please go through the [`CONTRIBUTING.md` file](https://github.com/HarshKapadia2/git_basics/blob/main/CONTRIBUTING.md#parsing) properly and always test the final site before committing and pushing. This will ensure that the PR will get merged quicker.

HarshKapadia2 commented:
Also, the CI/CD workflow has been modified again, so I would recommend [updating your branch with the latest commits](https://harshkapadia2.github.io/git_basics/#_sync_a_fork_with_its_original_repo).

shefali12-ab commented:
@HarshKapadia2 I have made the required, i need help please tell me how to perform testing of website. as I have read contributing.md and there it was written to generate website in build directory and run sh build.sh command . i didn't understand like where to  run these commands( i am doing it like opening new terminal in open_source_programs file and typed sh build.sh command )but it didn't worked .please guide me.

HarshKapadia2 commented:
Please remove all the logos as well. They are not required.

HarshKapadia2 commented:
It seems that the Table of Contents (ToC) is missing in your output. Weird.

In the image below, please note how all programs are appearing in the ToC rather than just the heading 'Open Source Programs'. Please look into your heading levels (no. of `=`). There should only be one level three heading (`===`) in a file. All the others should be between level 4 to 6.

<img src="https://user-images.githubusercontent.com/50140864/137638975-5f8219b1-9a7b-4a70-b91e-8adddc443fe9.png" width="70%" />

Please change the title of the 'Official Website' table column to 'Links'.

Some of the empty table cells have a single dash while other have three hyphens. Please make them all long dashes.

The middle column line of the table is thicker than the rest. Why is that? Please make it similar to the others.

The main branch has been updated with some code, so please update your branch.

HarshKapadia2 commented:
Any updates, @shefali12-ab?
